### PR TITLE
[Rubocop] Layout Cop Offenses

### PR DESCRIPTION
**Pull request by colinross**
Begins to address #1202 

This is just the first stack of the 'Layout' cops corrected. I broke up the changes into a set of commits that should be able to easily digest. 

I also decided to not move on the `Layout/IndentHeredoc` cop, since I was a bit unsure how you would prefer to have HEREDOC spacing enforced (there is no rubocop default). I didn't notice anything in the greater thoughtbot policy on this either. See: https://github.com/rubocop-hq/rubocop/blob/master/manual/cops_layout.md#layoutindentheredoc

### CODE REVIEW

**Comment by colinross** [comment]
Also, I'll be avoiding any @houndci-bot issues unless I _introduced_ them in order to not muck up the waters too much.

File: gemfiles/4.2.gemfile [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 source "https://rubygems.org"
 
+gem "activerecord", "~> 4.2.5.1"
```
Comment (composerinteralia) [inline gemfiles/4.2.gemfile]: The files in `gemfiles/` are generated by Appraisal, so we shouldn't edit them. Instead, let's tell RuboCop to exclude this directory.

**Comment by colinross** [comment]
Ignored the `gemfiles/`;  feel free to squash merge.

I'll 'tackle' the HEREDOCs in a later PR. Since we do have a hard dep on activesupport (in the gemspec) I'm leaning towards enforcing the active support style, which would be the `MESSAGE.strip_heredoc` style since it would be rude to bump the ruby version requirement to 2.3 just for enforcing squiggly style `<<~MESSAGE`.

**Comment by composerinteralia** [comment]
fb 5 won't officially support anything before Ruby 2.3, so the squiggly HEREDOC is a possibility.

**Pull request merged**